### ISSUE TITLE
Exact Permission for Connecting Log Analytics workspace

### ISSUE THREAD

**Issue by jayendranarumugam** [MicrosoftDocs/powerbi-docs#5095]
As per [this doc](https://github.com/MicrosoftDocs/powerbi-docs/blob/main/powerbi-docs/transform-model/log-analytics/desktop-log-analytics-overview.md) it mentioned a permission called 
`Power BI workspace owner`, or `Log Analytics workspace owner` . but there isn't any such role exist on Azure. What is the correct role required ? So far only the `Owner` role worked. But the owner is super admin kind of role and from the securit side we should not give and provide a lest permission as possible. So what is the exact permission is required in this case

**Comment by bishalgoswami** [comment]
Thanks for your comment, @jayendranarumugam. This issue has been assigned to the doc author for review.

**Comment by paulinbar** [comment]
Hi @jayendranarumugam, you might try checking out this article. Does this give you information you need? https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-powerbi

**Comment by jayendranarumugam** [comment]
@paulinbar I dont think so, the link you are referring is still the same wordings `Power BI workspace owner`, or `Log Analytics workspace owner`  but there isn't any such role exist on Azure. What is the correct role required ? So far only the Owner role worked. But the owner is super admin kind of role and from the securit side we should not give and provide a lest permission as possible. So what is the exact permission is required in this case

**Comment by paulinbar** [comment]
Hi @jayendranarumugam, are you sure that the information you need isn't in either the Prerequisites section here: https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-powerbi#prerequisites or the Required permissions section that follows it? Or in the docs that those sections link to?
